kubera/authentication/rest_test.go

```diff
@@ -0,0 +1,28 @@
+package rest
+
+import (
+	"log"
+	"testing"
+)
+
+var config Connection
+
+func init() {
+	fetchConnection, err := NewTokenFetcher()
+	if err != nil {
+		log.Fatalln(err)
+		return
+	}
+	log.Printf("\n UserName : %s ,\n Password : %s, \n Host : %s ", fetchConnection.Host, fetchConnection.UserName, fetchConnection.Host)
+	config = fetchConnection.Connection
```

Comment:
Don't use Token to fetch connection.
Use init func() to populate connection details from ENV

Reply:
Alternatively you may use sync.Once function to populate connection.
Remember we want to populate / fillup the connection . In this case we want to populate via ENV.


In following example I have made use of sync.Once
```go
type Connection struct {
  Hostname string
  Username string
  Password string
}

var connection Connection

func GetConnection() Connection {
  if connection.Hostname != "" {
    return connection
  }
  sync.Once(func(){
    connection = Connection{
      host: env.Get(KUBERA_HOST_NAME)
      user: env.Get(KUBERA_USER_NAME)
      pw: env.Get(KUBERA_PASSWORD)
    }
  })
  return connection
}
```